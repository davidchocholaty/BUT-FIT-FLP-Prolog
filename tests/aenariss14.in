S   S R
S a B a
B a B R
B c B R
B   D R
D   E R
E   F a
            aaacaa
