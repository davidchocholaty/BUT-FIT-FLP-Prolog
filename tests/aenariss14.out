S            aaacaa
 S           aaacaa
  S          aaacaa
   S         aaacaa
    S        aaacaa
     S       aaacaa
      S      aaacaa
       S     aaacaa
        S    aaacaa
         S   aaacaa
          S  aaacaa
           S aaacaa
            Saaacaa
            Baaacaa
            aBaacaa
            aaBacaa
            aaaBcaa
            aaacBaa
            aaacaBa
            aaacaaB 
            aaacaa D 
            aaacaa  E 
            aaacaa  Fa
